src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Automation/AbstractExtenderCATIDProvider.cs

```diff
@@ -21,41 +21,27 @@ internal abstract class AbstractExtenderCATIDProvider : IExtenderCATIDProvider
             // The latter issue we can do nothing about, however, to address the former, map these types to a truer form it makes 
             // it easier on implementors and maintainers of this to understand the objects we're talking about.
 
-            switch (extenderCATIDType)
+            return extenderCATIDType switch
```

Comment:
This is the only conversion from switch statement to switch expression that was a little odd. I tried a few different alignments and settled on sticking with the spirit of the original formatting, which aligned the right-handed comments. If there is another alignment that makes sense, suggest it. If we want to revert to the previous switch statement, I will do so.